assets/scss/core/_foundation.scss

```diff
@@ -168,3 +169,7 @@ h6 {
     @include font(body);
     font-weight: bold;
 }
+
+mark {
+    background-color: rgba($color-link, 0.20);
```

Comment:
Would also suggest making the font bold

Reply:
font-weight changes might make the UI jump around a bit, when words start breaking onto new lines.

Reply:
Since `white-space: nowrap;` is set on `.Tree` it doesn't seem like there's a risk of words breaking onto new lines, right?

Reply:
oh, i was not aware of that.

that... seems like bug that someone would report:
![image](https://user-images.githubusercontent.com/1892091/80633235-1bbc2f80-8a61-11ea-85bf-9783347586fd.png)

Reply:
I do not agree for the bold, it will indeed makes the text move as you type in the search field and I feel that's too disturbing. Also the default yellow is really ugly and doesn't really fit with the theme. I don't want to pick a color for every skin though, so that's why I went for a light variant of an existing color.

I made the color slightly more visible though. Also I decided to not tackle the white-space issue in this PR to avoid making it too large.